### DISCUSSION TITLE
MySQL connection is working, Query doesn't. I'm getting error No Module 'MySQLdb'

### DISCUSSION THREAD

**Discussion by AthanPal** [apache/superset#28553] [Q&A / Help]
Hello,

i have db connection and shcema and table schema... but query and datasets don't work V 4.0.1

![image](https://github.com/apache/superset/assets/31349946/bfe2d526-d558-4390-b142-f6f4e188d2b5)

pip freeze | grep sql
marshmallow-sqlalchemy==0.28.2
mysql-connector==2.2.9
mysql-connector-python==8.4.0
mysqlclient==2.2.4
pymssql==2.3.0
sqlglot==23.15.8
sqlparse==0.4.4

Best regards and thank you in advance for responding

**Comment by SanketMandhare** [comment]
I am also facing the same issue with mysql connection , is your issue resolved? or any leads on this.

<img width="959" alt="mysqldb_issue" src="https://github.com/user-attachments/assets/2ca9b875-e8eb-4f70-9726-9686ebf84ae3" />

**Comment by HWTsang** [comment]
Maybe you have installed less third-party libraries?
![image](https://github.com/user-attachments/assets/8ba939bc-fe50-4bd9-94f1-bd77f477b3bd)
superset4.1.1
